expression/expression.go

```diff
@@ -698,3 +709,155 @@ func wrapWithIsTrue(ctx sessionctx.Context, keepNull bool, arg Expression) (Expr
 	}
 	return FoldConstant(sf), nil
 }
+
+func getDatumBound(retType *types.FieldType, rType RoundingType) types.Datum {
+	if rType == Ceiling {
+		switch retType.Tp {
+		case mysql.TypeShort:
```

Comment:
Can we reuse the function of `GetMaxValue` in `feedback.go`?

Reply:
Yes, I'll try it.